FND/Places/Addresses.rdf

```diff
@@ -99,29 +120,457 @@
+	</owl:Class>
+	
+	<owl:Class rdf:about="&fibo-fnd-plc-adr;StreetAddress">
+		<rdfs:subClassOf rdf:resource="&fibo-fnd-plc-adr;AddressComponent"/>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-fnd-arr-arr;hasConstituent"/>
+				<owl:onClass rdf:resource="&fibo-fnd-plc-adr;SecondaryUnit"/>
+				<owl:maxQualifiedCardinality rdf:datatype="&xsd;nonNegativeInteger">1</owl:maxQualifiedCardinality>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-fnd-arr-arr;hasConstituent"/>
+				<owl:onClass rdf:resource="&fibo-fnd-plc-adr;StreetSuffix"/>
+				<owl:maxQualifiedCardinality rdf:datatype="&xsd;nonNegativeInteger">1</owl:maxQualifiedCardinality>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-fnd-arr-arr;hasConstituent"/>
+				<owl:onClass rdf:resource="&fibo-fnd-plc-adr;PostdirectionalSymbol"/>
+				<owl:maxQualifiedCardinality rdf:datatype="&xsd;nonNegativeInteger">2</owl:maxQualifiedCardinality>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-fnd-arr-arr;hasConstituent"/>
+				<owl:onClass rdf:resource="&fibo-fnd-plc-adr;PredirectionalSymbol"/>
+				<owl:maxQualifiedCardinality rdf:datatype="&xsd;nonNegativeInteger">2</owl:maxQualifiedCardinality>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-fnd-arr-arr;hasConstituent"/>
+				<owl:onClass rdf:resource="&fibo-fnd-plc-adr;PrimaryAddressNumber"/>
+				<owl:qualifiedCardinality rdf:datatype="&xsd;nonNegativeInteger">1</owl:qualifiedCardinality>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-fnd-arr-arr;hasConstituent"/>
+				<owl:onClass rdf:resource="&fibo-fnd-plc-adr;StreetName"/>
+				<owl:qualifiedCardinality rdf:datatype="&xsd;nonNegativeInteger">1</owl:qualifiedCardinality>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:label>street address</rdfs:label>
+		<skos:definition>index to a location that consists of a primary address number, predirectional, street name, suffix, postdirectional, and an optional secondary unit</skos:definition>
+		<fibo-fnd-utl-av:explanatoryNote>If the street address represents a single mailbox, residence, office, or other secondary unit, then it qualifies as a unique delivery address.</fibo-fnd-utl-av:explanatoryNote>
+	</owl:Class>
+	
+	<owl:Class rdf:about="&fibo-fnd-plc-adr;StreetName">
+		<rdfs:subClassOf rdf:resource="&fibo-fnd-plc-adr;AddressComponent"/>
+		<rdfs:subClassOf rdf:resource="&lcc-lr;Identifier"/>
+		<rdfs:label>street name</rdfs:label>
+		<skos:definition>identifier for a street in some context (e.g., city, municipality, geographic region)</skos:definition>
```

Comment:
Ok, removed that, but added hasTag for the text

Reply:
Note that we may want to change 'hasTag' in LCC to be either a string or langString ... so that we can allow for multilingual tags.  The address line properties are defined to use rdfs:Literal for that reason.

Reply:
I agree with changing hasTag for the reason you give